examples/pbf2d.py

```diff
@@ -194,7 +181,7 @@ def find_particle_neighbors():
 
 
 @ti.kernel
-def compute_lambdas():
+def substep():
```

Comment:
Actually, would you not merge them into a single kernel? I wrote the names to make the computation stages obvious.

Reply:
Why not add some comments per-offload instead? I just want to export them to [Taichi.js](https://taichi-dev.github.io/taichi.js) easier..

Reply:
I see. Sorry i don't know much about js, is there a limitation for how many kernels we can have there?

Reply:
Just hard to seek out the launch order..